python/pyarrow/tests/test_flight.py

```diff
@@ -1993,6 +1993,11 @@ def test(read_all):
         descriptor = flight.FlightDescriptor.for_command(b"echo")
         writer, reader = client.do_exchange(descriptor)
         test(reader.read_all)
+        try:
+            writer.close()
+        except Exception:
+            # Silence the Cancelled/Interrupt exception
```

Comment:
If we only want to silence a certain class of exceptions, can the `except` clause be more specific?

Reply:
Good point, listed out KeyboardInterrupt/FlightCancelledError